src/nfs-ganesha/handle.c

```diff
@@ -1,4 +1,4 @@
-﻿/*
```

Comment:
What was the change in the first line?

Reply:
Probably need to check all the files and remove all these BOM's, but that's a separate PR.